{{ cookiecutter.hosting_github_repo_name }}/python/event-gather-config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "google_credentials_file": "google-creds.json",
+  "gcs_bucket_name": "",
+  "caption_new_speaker_turn_pattern": "",
+  "caption_confidence": null
```

Comment:
I don't these should actually be set at all. If you set them they will override the defaults. `null` is fine because we do a check for `None` later but `""` (empty string) would set the caption new speaker turn to just any line I think?

Reply:
Sounds good, I'll just set these values to `null` so that the fields will at least be present in the file so it'll be easier for users to know that they can change them.